Src/LiquidProjections.NHibernate/NHibernateProjector.cs

```diff
@@ -81,6 +81,11 @@ public string StateKey
         }
 
         /// <summary>
+        /// A custom policy that will be executed when projecting a batch of transactions fails. This can be used to retry the batch.
+        /// </summary>
+        public ExceptionPolicy ExceptionPolicy { get; set; }

```

Comment:
I think calling this stuff a `RetryPolicy` makes it more explicit that it is only called after an exception occurs.